root/us-east-1/notifications/costs_tools_monitoring.tf

```diff
@@ -13,11 +16,21 @@ module "notify_costs" {
     }
   }
 
-  # Policy
+  # Access policy document
   sns_topic_policy_json = join("", data.aws_iam_policy_document.aws_sns_topic_policy.*.json)
 
 }
 
+# Subscribing a list of email addresses to SNS topic
+resource "aws_sns_topic_subscription" "topic_email_subscription" {
+  count                  = length(var.costs_email_addresses)
```

Comment:
@lgallard thanks for pointing this out, I agree with it and think we can address this in a future PR, for the moment we'll only use `info@binbash.com.ar` which is an already publicly exposed email address.

note: Luis just in case Jose's user is @joseapeinado  :)